src/FlightDisplay/FlightDisplayView.qml

```diff
@@ -469,8 +469,8 @@ QGCView {
             z:                  _panel.z + 4
             title:              qsTr("Fly")
             maxHeight:          (_flightVideo.visible ? _flightVideo.y : parent.height) - toolStrip.y
-            buttonVisible:      [ _guidedController.showTakeoff || !_guidedController.showLand, _guidedController.showLand && !_guidedController.showTakeoff, true, true, true, _guidedController.smartShotsAvailable ]
-            buttonEnabled:      [ _guidedController.showTakeoff, _guidedController.showLand, _guidedController.showRTL, _guidedController.showPause, _anyActionAvailable, _anySmartShotAvailable ]
+            buttonVisible:      [ QGroundControl.settingsManager.appSettings.useChecklist.rawValue, _guidedController.showTakeoff || !_guidedController.showLand, _guidedController.showLand && !_guidedController.showTakeoff, true, true, true, _guidedController.smartShotsAvailable ]
+            buttonEnabled:      [ QGroundControl.settingsManager.appSettings.useChecklist.rawValue, _guidedController.showTakeoff, _guidedController.showLand, _guidedController.showRTL, _guidedController.showPause, _anyActionAvailable, _anySmartShotAvailable ]
```

Comment:
Would be better to move QGroundControl.settingsManager.appSettings.useChecklist.rawValue into a private bool like _anyActionAvailable and then reference that to remove duplication of such a long reference.

Reply:
@philipoe If you don't have time to do this right now. I can merge this in and then I'll take care of this change myself. Let me know. Don't want to have this sitting here too long. Then it gets out of sync and you run into nasty merge conflicts.

Reply:
Sorry missed that! Looking...